pkgs/applications/networking/remote/dayon/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ant
+, jdk
+, jre
+, makeWrapper
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dayon";
+  version = "11.0.7";
+
+  src = fetchFromGitHub {
+    owner = "RetGal";
+    repo = "dayon";
+    rev = "v${version}";
+    hash = "sha256-3TbJVM5po4aUAOsY7JJs/b5tUzH3WGnca/H83IeMQ2s=";
+  };
+
+  postPatch = ''
+    substituteInPlace resources/deb/dayon_assisted.desktop resources/deb/dayon_assistant.desktop \
+      --replace "Exec=/usr/bin" "Exec=$out/bin" \
+      --replace "Icon=/usr/bin/dayon/dayon.png" "Icon=dayon.png"
```

Comment:
You could propose a patch to upstream to make it more portable for everyone.

Reply:
You mean using basenames for both?

Reply:
Whatever helps it to work not only in `/usr/bin`. I don't know the solution.

Reply:
I proposed a patch upstream https://github.com/RetGal/Dayon/pull/66